pkg/verifier/codeowner.go

```diff
@@ -102,8 +101,16 @@ func ValidateCodeownerFile(p providers.Provider, filename string) (bool, error)
 	}
 	valid := true
 	for _, c := range codeowners {
-		// This doesn't works for ! patterns
-		if files, _, err := glob.Glob([]string{c.Path}); err != nil || len(files) < 1 {
+		currentDir, _ := os.Getwd()
+		files, _ := FilePathWalkDir(currentDir)
+		fileMatches := false
+		for _, file := range files {
```

Comment:
since we are just looking for a match. It can be simplified to:

```golang
// Search for a single match
for idx := 0; idx < len(files) && !filesMathces; idx++ {
   // remove current dir from filepath to use regex properlly.
    file := strings.Replace(files[idx], currentDir, "", 1)
    filesMatches = c.MatchesPath(file)
}
```

What do you think?